portfolio/src/main/webapp/script.js

```diff
@@ -174,6 +174,21 @@ function deleteData() {
   });
 }
 
+/**
+ * Loads the blobstore image storage and shows the form to submit an image.
+ */
+function loadBlobstoreAndForm() {
+  fetch('/blobstore-upload-url')
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('image-submit');
+        messageForm.action = imageUploadUrl;
```

Comment:
When I upload a photo on the deployed version, I get a 500 server error. I think it might be because you return response.text() which exits the function and the form action is never set?

Reply:
Oh yikes! This was working for me earlier, but you're right, I get the same error now. I'll have to debug this.

Reply:
I'm seeing that a lot of people apparently have issues with Blobstore once it's deployed, and you have to change some things to make it work on the deployed server as well as the dev one. I'll work on that and re-push.

Reply:
Actually, I'm able to see the upload form, so I think it's probably in the /image-handler